docs/docs/external/getting-started/enabling-weaver-network/fabric.md

```diff
@@ -0,0 +1,398 @@
+
+      The rest of the code ought to be self-explanatory. Values are hardcoded for explanation purposes, but you can refactor the above code by reading view addresses corresponding to chaincode invocations from a configuration file.
+
+      You also need to add the following dependency to the `dependencies` section of your application's `package.json` file:
+      ```
+      "@hyperledger-labs/weaver-fabric-interop-sdk": "latest",
+      ```
+      (Or check out the [package website](https://github.com/hyperledger-labs/weaver-dlt-interoperability/packages/888424) and select a different version.)
+
+Create a personal access token with `read:packages` access in github in order to use modules published in github packages. Refer [Creating a Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) for help.
+      Before you run `npm install` to fetch the dependencies, make sure you create a [personal access token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) with `read:packages` access in Github. Create an `.npmrc` file in the same folder as the `package.json` with the following contents:
+      ```
+      registry=https://npm.pkg.github.com/hyperledger-labs
+      //npm.pkg.github.com/:_authToken=<personal-access-token>
+      ```
+      Replace `<personal-access-token>` in this file with the token you created in Github.
+    - _Asset exchange_: _TBD_
+
+### Pre-Configuration
+
+No changes are required in your network's pre-configuration process for Weaver enablement.
+
+Typically, pre-configuration involves generating:
+- _Channel artifacts_: orderer genesis block, channel transaction, and anchor peer configurations from a `configytx.yaml` file (using Fabric's `configtxgen` tool)
```

Comment:
typo: `configytx.yaml`? (optional, not sure if it was intentional)

Reply:
Fixed